packages/react-core/src/components/Wizard/examples/Wizard.md

```diff
@@ -497,15 +497,15 @@ class ProgressiveWizard extends React.Component {
             isChecked={getStartedStepRadio === 'Create'}
             onChange={(_, event) => this.setState({ getStartedStepRadio: event.currentTarget.value })}
             label="Create a new thing"
-            name="radio-step-start"
+            name="radio-step-start-1"
             id="radio-step-start-1"
           />{' '}
           <Radio
             value="Update"
             isChecked={getStartedStepRadio === 'Update'}
             onChange={(_, event) => this.setState({ getStartedStepRadio: event.currentTarget.value })}
             label="Update an existing thing"
-            name="radio-step-start"
+            name="radio-step-start-2"
```

Comment:
I think when you're defining a radio group the `name` attribute is what relates the inputs in that group together. The keyboard interaction on radio has a slightly different behavior where the first radio in the group will receive focus, and then you can use arrow keys to shift focus to the other radios. So this would be like setting two radio groups.

Agree the standard tab focus is nice, but I think it might actually be behaving as expected? The best way to see is check out the Form Controls rotor menu in VoiceOver, they'll both mention that the radios are "1 of 1" in the group after it announces the name. Hope it helps!

Reply:
Ah, thank you for this catch! 🙂